website/content/labs/lab1.md

```diff
@@ -30,30 +43,34 @@ Log into https://mit.cs.uchicago.edu/. Under "Your projects", you should see an
   cd cs220
   ```
 
-* Inside that folder, create a file called `test.txt` with your full name.
+* Inside that folder, create a file called `README.md` with your full name.
```

Comment:
Instead of `with your full name`, maybe we should change the wording to something along the lines of `and add your full name to the file`. Really minor, but the original wording seemed a bit vague to me when I read it for the first time.

Reply:
Pushed a fix for this.